assembly/onpremises-ide-packaging-war-platform-api/src/main/webapp/WEB-INF/classes/codenvy/codenvy-api-configuration.properties

```diff
@@ -56,3 +56,5 @@ che.stacks.images.storage=${catalina.base}/stacks/images
 stop.workspace.scheduler.period=60
 
 machine.ssh.connection_timeout_ms=3000
+
+system_ram.limit.check.scheduler.period=60
```

Comment:
Yes, the name can be improved. @vinokurig take into account that we use convention:
- dots separate subresources.
- underscores unites complex terms
- property with number value must include measurement unit in last term separated by uinderscore
 E.g.  machine.ssh.connection_timeout_ms=3000 - it is connected to machine, ssh implementation of machine, described connection timeout in milliseconds